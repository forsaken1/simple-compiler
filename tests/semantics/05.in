void main(int i) {
	i;
}