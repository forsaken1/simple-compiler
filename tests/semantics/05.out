void function (main)
|
+---(i)
|
+---{stmt}
    |
    +---(i)